Editor/Helpers/CinemachineDefaultMouseInput.cs

```diff
@@ -0,0 +1,92 @@
+        
+        /// <summary>
+        /// Initialize-on-demand singleton.
+        /// </summary>
+        /// <returns>Initialized instance</returns>
+        public static CinemachineDefaultMouseInput GetInstance() {
+            if (s_Instance == null) {
+                s_Instance = new CinemachineDefaultMouseInput();
+            }
+            return s_Instance;
+        }
+
+        CinemachineDefaultMouseInput()
+        {
+            var inputActionAsset = AssetDatabase.LoadAssetAtPath<InputActionAsset>("Packages/com.unity.inputsystem/" +
+                "InputSystem/Plugins/PlayerInput/DefaultInputActions.inputactions");
+
+            using var inputActions = inputActionAsset.GetEnumerator();
+            var hasNext = true;
+            while (hasNext)
+            {
+                if (inputActions.Current != null &&
+                    inputActions.Current.ToString() == "Player/Look[/Mouse/delta,/Pen/delta]")
+                {
+                    var look = inputActions.Current;
+                    m_InputActionReference = InputActionReference.Create(look);
+                    m_InputActionReference.name = "Generic Look";
+                    break;
+                }
+
+                hasNext = inputActions.MoveNext();
+            }
+        }
+        InputActionReference GetInputActionReference()
+        {
+            return m_InputActionReference;
+        }
+
+        static GUIContent s_InputProviderAddLabel = new GUIContent(
+            "Add CinemachineInputProvider", "Adds CinemachineInputProvider to this vcam, if it does not have one already, " +
+            "enabling the vcam to read input from Input Actions. By default, a simple mouse XY input action is added.");
+        
+        /// <summary>
+        /// Adds an information sign and a button that adds adds CinemachineInputProvider component to the vcam with a
+        /// default look control (XY axis), if the gameobject has at least one component or extension that requires
+        /// input and the vcam does not already have a CinemachineInputProvider component. For a component or extension
+        /// to require input, the component or extension needs to override RequiresUserInput in CinemachineComponentBase or
+        /// CinemachineExtension respectively.
+        /// <seealso cref="CinemachineVirtualCameraBaseEditor{T}"/>
+        /// </summary>
+        /// <param name="gameObject">The gameObject to which we'd like to add the CinemachineInputProvider
+        /// via a Button interface</param>
+        public void InputProviderButton(GameObject gameObject)
+        {
+            var inputProvider = gameObject.GetComponent<CinemachineInputProvider>();
+            if (inputProvider != null) return;
+            
+            EditorGUILayout.HelpBox("InputSystem package is installed, but it is not used to control this vcam.", 
+                MessageType.Info);
+            GUILayout.BeginHorizontal();
+            GUILayout.FlexibleSpace();
```

Comment:
Layout still looks a little messy to me.  What do you think of this option?  Had to edit the button text to make it fit.

![image](https://user-images.githubusercontent.com/6424658/112551591-a3a4d980-8d97-11eb-9119-a69c511a607b.png)

          EditorGUILayout.HelpBox("The InputSystem package is installed, but it is not used to control this vcam.", 
              MessageType.Info);
          var rect = EditorGUILayout.GetControlRect(true);
          rect.x += EditorGUIUtility.labelWidth; 
          rect.width -= EditorGUIUtility.labelWidth;
          if (GUI.Button(rect, s_InputProviderAddLabel))
          {
              inputProvider = Undo.AddComponent<CinemachineInputProvider>(gameObject);
              inputProvider.XYAxis = GetInputActionReference();
          }